libs/Model/Model.cpp

```diff
@@ -52,16 +50,20 @@ void Model::render(ShaderProgram& shaderProgram)
 
 	assert(m_Obj_Model && "NO MODEL TO RENDER");
 	// bind texture
-	for (int i = 0; i < m_textures.size(); ++i)
-	{
-		m_textures[i]->Bind(i);
-	}
+	
+	m_texture->Bind(0);
```

Comment:
it looks like texture used to be a vector and is now a scalar - why the change here?

Reply:
Previously the models used to support multiple textures. But the shader code only links the first texture textures[0]. So, for now I removed the multitexture support.

Reply:
could you add a comment with that info?

Reply:
done